assets/airports/cyow.json

```diff
@@ -0,0 +1,1769 @@
+        "MERKO": ["N45d27m07.00", "W075d43m58.00"],
+        "MIVOK": ["N44d21m60.00", "W077d35m34.00"],
+        "MODON": ["N45d17m57.00", "W075d56m19.00"],
+        "NOVUX": ["N45d12m57.00", "W075d40m88.00"],
+        "OBTAX": ["N45d41m68.00", "W073d16m10.00"],
+        "OLIGO": ["N45d29m39.00", "W076d15m00.00"],
+        "OMTUN": ["N45d22m69.00", "W075d59m11.00"],
+        "ONBAB": ["N45d28m92.00", "W075d36m08.00"],
+        "ONDOB": ["N45d41m62.00", "W076d26m25.00"],
+        "PETLU": ["N45d30m81.00", "W075d50m69.00"],
+        "RADEN": ["N45d55m92.00", "W076d01m58.00"],
+        "RAFNO": ["N45d25m55.00", "W075d52m64.00"],
+        "REVAL": ["N45d27m69.00", "W075d39m57.00"],
+        "SAVAL": ["N44d54m25.00", "W074d42m02.00"],
+        "SEBEG": ["N45d16m37.00", "W075d54m01.00"],
+        "SENNZ": ["N45d20m57.00", "W075d49m50.00"],
+        "SMARE": ["N46d19m56.00", "W078d09m82.00"],
+        "TAPVO": ["N44d52m55.00", "W074d58m23.00"],
+        "TEFLY": ["N45d24m43.00", "W075d27m71.00"],
+        "TEXEN": ["N45d12m53.00", "W075d27m56.00"],
+        "THURO": ["N45d33m42.00", "W074d54m47.00"],
+        "TIDUL": ["N45d15m41.00", "W075d21m02.00"],
+        "TILOG": ["N45d09m12.00", "W075d48m52.00"],
+        "TOXIP": ["N45d55m76.00", "W077d04m07.00"],
+        "TUKIR": ["N45d15m12.00", "W076d14m34.00"],
+        "ULAGU": ["N45d11m31.00", "W075d43m66.00"],
+        "URLAG": ["N45d21m55.00", "W075d20m89.00"],
+        "VILRO": ["N45d36m95.00", "W072d42m93.00"],
+        "VINRA": ["N45d16m49.00", "W075d58m57.00"],
+        "VISOL": ["N45d13m34.00", "W075d52m35.00"],
+        "VOLAG": ["N45d08m51.00", "W075d31m75.00"],
+        "WATTO": ["N45d15m97.00", "W075d22m22.00"],
+        "XOLTI": ["N45d28m65.00", "W075d31m52.00"],
+        "YUL"  : ["N45d36m94.00", "W073d58m26.00"]
+    },
+    "runways": [
+        {
+            "name": ["04", "22"],
+            "end": [
+                ["N45d19m36.00", "W075d41m20.50", "347ft"],
+                ["N45d20m03.10", "W075d41m02.30", "335ft"]
+            ],
+            "ils": [false, false]
+        },
+        {
+            "name": ["07", "25"],
+            "end": [
+                ["N45d18m47.80", "W075d40m16.80", "369ft"],
+                ["N45d19m30.20", "W075d38m42.40", "362ft"]
+            ],
+            "ils": [true, false]
+        }, 
+        {
+            "name": ["14", "32"],
+            "end": [
+                ["N45d19m37.30", "W075d41m10.40", "140ft"],
+                ["N45d18m38.80", "W075d39m17.70", "367ft"]
+            ],
+            "ils": [false, true]
+        }
+    ],
+    "sids": {
+        "CYOW4": {
+            "icao": "CYOW4",
+            "name": "Ottawa Four",
+            "rwy": {
+                "04": ["#039"],
+                "07": ["#071"],
+                "14": ["#140"],
+                "22": ["#219"],
+                "25": ["#251"],
+                "32": ["#320"]
+            },
+            "exitPoints": {
+                "APLOV": ["APLOV"],
```

Comment:
Same issue as in #117, Since these fixes aren't charted I'm not sure this is the way we should be adding them. However, I can't think of any other ways of doing it, so honestly I don't know  :man_shrugging: @erikquinn?

Reply:
Well those were off FlightAware, so I guess I won't put SIDs in

Reply:
I wouldn't get rid of the SID, we still want the aircraft to fly those headings after departure.

Reply:
Seems like a sensible enough way to do it to me